practice/00-HTML-CSS-basics/V.html

```diff
@@ -2,6 +2,7 @@
 <html>
```

Comment:
Please review the full code of this file. I see the tags are not closed and also not well indented . As an example, I see the tag < body> does not close with < / body>